### ISSUE TITLE
Process hang in systray 

### ISSUE THREAD

**Issue by Elocin-Anagram** [obsproject/obs-studio#3569]
## Platform

AMD Ryzen 9 3900X - OK! / Radeon RX 5700 XT - OK!

Operating system and version:
OBS Studio version: 26.0.2

## Expected Behavior

When I close obs it should close and not hang in my systray.
## Current Behavior

 I cannot open it again till it stops hanging.

<details>
<summary>OBS Log</summary>

```
00:46:08.355: CPU Name: AMD Ryzen 9 3900X 12-Core Processor            
00:46:08.355: CPU Speed: 3793MHz
00:46:08.355: Physical Cores: 12, Logical Cores: 24
00:46:08.355: Physical Memory: 32716MB Total, 21810MB Free
00:46:08.355: Windows Version: 10.0 Build 19041 (release: 2004; revision: 508; 64-bit)
00:46:08.355: Running as administrator: true
00:46:08.355: Aero is Enabled (Aero is always on for windows 8 and above)
00:46:08.355: Windows 10 Gaming Features:
00:46:08.355: 	Game Bar: On
00:46:08.355: 	Game DVR: On
00:46:08.355: 	Game DVR Background Recording: Off
00:46:08.358: Sec. Software Status:
00:46:08.359: 	Malwarebytes: enabled (AV)
00:46:08.359: 	Microsoft Defender Antivirus: disabled (AV)
00:46:08.359: 	Windows Firewall: enabled (FW)
00:46:08.360: Current Date/Time: 2020-10-07, 00:46:08
00:46:08.360: Browser Hardware Acceleration: true
00:46:08.360: Portable mode: false
00:46:08.599: OBS 26.0.2 (64-bit, windows)
00:46:08.599: ---------------------------------
00:46:08.617: ---------------------------------
00:46:08.617: audio settings reset:
00:46:08.617: 	samples per sec: 48000
00:46:08.617: 	speakers:        2
00:46:08.627: ---------------------------------
00:46:08.627: Initializing D3D11...
00:46:08.627: Available Video Adapters: 
00:46:08.631: 	Adapter 0: AMD Radeon RX 5700 XT
00:46:08.631: 	  Dedicated VRAM: 4253294592
00:46:08.631: 	  Shared VRAM:    4268019712
00:46:08.631: 	  PCI ID:         1002:731f
00:46:08.631: 	  Driver Version: 27.20.12029.1000
00:46:08.631: 	  output 0: pos={0, 0}, size={1440, 900}, attached=true, refresh=60, name=CNDLCD
00:46:08.631: 	  output 1: pos={-544, -768}, size={1360, 768}, attached=true, refresh=60, name=ELEFW328
00:46:08.631: 	  output 2: pos={-1680, 0}, size={1680, 1050}, attached=true, refresh=60, name=HP w2207
00:46:08.633: Loading up D3D11 on adapter AMD Radeon RX 5700 XT (0)
00:46:08.659: D3D11 loaded successfully, feature level used: b000
00:46:08.659: DXGI increase maximum frame latency success
00:46:08.659: D3D11 GPU priority setup success
00:46:09.036: ---------------------------------
00:46:09.036: video settings reset:
00:46:09.036: 	base resolution:   1440x900
00:46:09.036: 	output resolution: 1280x720
00:46:09.036: 	downscale filter:  Bicubic
00:46:09.036: 	fps:               30/1
00:46:09.036: 	format:            NV12
00:46:09.036: 	YUV mode:          709/Partial
00:46:09.037: NV12 texture support enabled
00:46:09.042: Audio monitoring device:
00:46:09.042: 	name: Default
00:46:09.042: 	id: default
00:46:09.042: ---------------------------------
00:46:09.092: [CoreAudio encoder]: CoreAudio AAC encoder not installed on the system or couldn't be loaded
00:46:09.111: Failed to load 'en-US' text for module: 'decklink-ouput-ui.dll'
00:46:09.409: [AMF] Version 2.7.0 loaded (Compiled: 1.4.14.0, Runtime: 1.4.17.0, Library: 1;4;17;0;20.20.29.01;202009092047;CL#2169548).
00:46:09.495: [AMF] [Capability Manager] Testing Direct3D 11 Adapter 'AMD Radeon RX 5700 XT (VEN_1002/DEV_731f/SUB_23131458/REV_00c1)':
00:46:09.495:   H264/AVC: Supported
00:46:09.495:   H265/HEVC: Supported
00:46:09.495: 
00:46:09.627: [obs-browser]: Version 2.8.7
00:46:09.673: [obs-ndi] hello ! (version 4.9.0)
00:46:09.673: [obs-ndi] Trying 'C:\Program Files\NewTek\NDI 4 Tools\Runtime'
00:46:09.673: [obs-ndi] Found NDI library at 'C:/Program Files/NewTek/NDI 4 Tools/Runtime/Processing.NDI.Lib.x64.dll'
00:46:09.724: [obs-ndi] NDI runtime loaded successfully
00:46:09.946: [obs-ndi] NDI library initialized successfully (NDI SDK WIN64 01:30:53 Sep  4 2020 4.5.3.0)
00:46:10.620: [obs-websocket] you can haz websockets (version 4.8.0)
00:46:10.620: [obs-websocket] qt version (compile-time): 5.10.1 ; qt version (run-time): 5.10.1
00:46:10.621: [obs-websocket] module loaded!
00:46:10.686: [tuna] Loading v2020.04.12 16:18
00:46:10.686: [tuna] libobs version 26.0.2 is invalid. Tuna expects 25.0.4 for VLC sources to work
00:46:10.686: [tuna] User force enabled VLC support
00:46:10.789: [tuna] Loaded libVLC. VLC source support enabled
00:46:10.826: [tuna] Registered Spotify (id: spotify)
00:46:10.826: [tuna] Registered MPD (id: mpd)
00:46:10.826: [tuna] Registered VLC (id: vlc)
00:46:10.826: [tuna] Registered Window Title (id: window)
00:46:10.826: [tuna] Loaded 1 outputs
00:46:10.827: [tuna] Couldn't move existing cover to temp file
00:46:10.827: [tuna] Couldn't move placeholder cover
00:46:10.833: VLC found, VLC video source enabled
00:46:10.860: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
00:46:10.861: No blackmagic support
00:46:10.895: ---------------------------------
00:46:10.895:   Loaded Modules:
00:46:10.895:     win-wasapi.dll
00:46:10.895:     win-openvr.dll
00:46:10.895:     win-mf.dll
00:46:10.895:     win-dshow.dll
00:46:10.895:     win-decklink.dll
00:46:10.895:     win-capture.dll
00:46:10.895:     vlc-video.dll
00:46:10.895:     tuna.dll
00:46:10.895:     text-freetype2.dll
00:46:10.895:     rtmp-services.dll
00:46:10.895:     obs-x264.dll
00:46:10.895:     obs-websocket.dll
00:46:10.895:     obs-vst.dll
00:46:10.895:     obs-transitions.dll
00:46:10.896:     obs-text.dll
00:46:10.896:     obs-qsv11.dll
00:46:10.896:     obs-outputs.dll
00:46:10.896:     obs-ndi.dll
00:46:10.896:     obs-filters.dll
00:46:10.896:     obs-ffmpeg.dll
00:46:10.896:     obs-browser.dll
00:46:10.896:     image-source.dll
00:46:10.896:     frontend-tools.dll
00:46:10.896:     enc-amf.dll
00:46:10.896:     decklink-ouput-ui.dll
00:46:10.896:     coreaudio-encoder.dll
00:46:10.896: ---------------------------------
00:46:10.896: ==== Startup complete ===============================================
00:46:10.911: All scene data cleared
00:46:10.911: ------------------------------------------------
00:46:11.540: [Media Source 'Highlight My Message']: settings:
00:46:11.540: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Sub chat Alert sounds/Navi Hey.mp3
00:46:11.540: 	input_format:            (null)
00:46:11.540: 	speed:                   100
00:46:11.540: 	is_looping:              no
00:46:11.540: 	is_hw_decoding:          no
00:46:11.540: 	is_clear_on_media_end:   yes
00:46:11.540: 	restart_on_activate:     yes
00:46:11.540: 	close_when_inactive:     no
00:46:11.555: [Media Source 'Truth or Dare']: settings:
00:46:11.555: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/Truth Or DARE/Truth or dare.mp3
00:46:11.555: 	input_format:            (null)
00:46:11.555: 	speed:                   100
00:46:11.555: 	is_looping:              no
00:46:11.555: 	is_hw_decoding:          no
00:46:11.555: 	is_clear_on_media_end:   yes
00:46:11.555: 	restart_on_activate:     yes
00:46:11.555: 	close_when_inactive:     no
00:46:11.560: [Media Source 'Change Avatar']: settings:
00:46:11.560: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/Change avatar/Change Avatar.mp3
00:46:11.560: 	input_format:            (null)
00:46:11.560: 	speed:                   100
00:46:11.560: 	is_looping:              no
00:46:11.560: 	is_hw_decoding:          no
00:46:11.560: 	is_clear_on_media_end:   yes
00:46:11.560: 	restart_on_activate:     yes
00:46:11.560: 	close_when_inactive:     no
00:46:11.565: [Media Source 'Chan pts End Stream']: settings:
00:46:11.565: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/End me/End Me.ogg
00:46:11.565: 	input_format:            (null)
00:46:11.565: 	speed:                   100
00:46:11.565: 	is_looping:              no
00:46:11.565: 	is_hw_decoding:          no
00:46:11.565: 	is_clear_on_media_end:   yes
00:46:11.565: 	restart_on_activate:     yes
00:46:11.565: 	close_when_inactive:     no
00:46:11.570: [Media Source 'Owo']: settings:
00:46:11.570: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/OwO/Processed/OwO.webm
00:46:11.570: 	input_format:            (null)
00:46:11.570: 	speed:                   100
00:46:11.570: 	is_looping:              no
00:46:11.570: 	is_hw_decoding:          no
00:46:11.570: 	is_clear_on_media_end:   yes
00:46:11.570: 	restart_on_activate:     yes
00:46:11.570: 	close_when_inactive:     no
00:46:11.575: [Media Source 'Navi Eep Sub chat notification']: settings:
00:46:11.575: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Plug-ins/lioranboard/lioranboard/LioranBoard Receiver(PC)/sound/Navi Eep 2.ogg
00:46:11.575: 	input_format:            (null)
00:46:11.575: 	speed:                   100
00:46:11.575: 	is_looping:              no
00:46:11.575: 	is_hw_decoding:          no
00:46:11.575: 	is_clear_on_media_end:   yes
00:46:11.575: 	restart_on_activate:     yes
00:46:11.575: 	close_when_inactive:     no
00:46:11.580: [Media Source 'Bonk Me']: settings:
00:46:11.580: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/Bonk Me/Processed/Bonk Me.webm
00:46:11.580: 	input_format:            (null)
00:46:11.580: 	speed:                   100
00:46:11.580: 	is_looping:              no
00:46:11.580: 	is_hw_decoding:          no
00:46:11.580: 	is_clear_on_media_end:   yes
00:46:11.580: 	restart_on_activate:     yes
00:46:11.580: 	close_when_inactive:     no
00:46:11.590: [Media Source 'ASMR']: settings:
00:46:11.590: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/Alerts/Channel Point Memes/ASMR/Processed/ASMR.webm
00:46:11.590: 	input_format:            (null)
00:46:11.590: 	speed:                   100
00:46:11.590: 	is_looping:              no
00:46:11.590: 	is_hw_decoding:          no
00:46:11.590: 	is_clear_on_media_end:   yes
00:46:11.590: 	restart_on_activate:     yes
00:46:11.590: 	close_when_inactive:     no
00:46:11.613: [Media Source 'COUNTDOWN']: settings:
00:46:11.613: 	input:                   C:/Users/tarap/Videos/1- OBS/Stream Assets/VIDEO/9 SEC COUNTDOWN.mov
00:46:11.613: 	input_format:            (null)
00:46:11.613: 	speed:                   100
00:46:11.613: 	is_looping:              no
00:46:11.613: 	is_hw_decoding:          no
00:46:11.613: 	is_clear_on_media_end:   yes
00:46:11.613: 	restart_on_activate:     yes
00:46:11.613: 	close_when_inactive:     no
00:46:11.672: [duplicator-monitor-capture: 'Display Capture'] update settings:
00:46:11.672: 	display: 1 (1440x900)
00:46:11.672: 	cursor: false
00:46:11.695: [Media Source 'Paul Adey LOL']: settings:
00:46:11.695: 	input:                   C:/Users/tarap/Music/06 LOL (Living Our Lives).mp3
00:46:11.695: 	input_format:            (null)
00:46:11.695: 	speed:                   100
00:46:11.695: 	is_looping:              no
00:46:11.695: 	is_hw_decoding:          no
00:46:11.695: 	is_clear_on_media_end:   yes
00:46:11.695: 	restart_on_activate:     yes
00:46:11.695: 	close_when_inactive:     no
00:46:11.749: WASAPI: Device 'Speakers (Scarlett Solo USB)' [44100 Hz] initialized
00:46:11.783: adding 42 milliseconds of audio buffering, total audio buffering is now 42 milliseconds (source: Desktop Audio)
00:46:11.783: 
00:46:12.006: WASAPI: Device 'Microphone (Antlion Wireless Microphone)' [48000 Hz] initialized
00:46:12.025: WASAPI: Device 'Microphone (Scarlett Solo USB)' [44100 Hz] initialized
00:46:12.114: [obs-ndi] started A/V threads for source 'FALCON (fx)'
00:46:12.114: [obs-ndi] A/V thread for 'falcon' started
00:46:12.114: [obs-ndi] started A/V threads for source 'FALCON (Falcon OBS Preview)'
00:46:12.115: [obs-ndi] A/V thread for 'f2' started
00:46:12.115: [obs-ndi] started A/V threads for source 'FALCON (f3)'
00:46:12.115: [obs-ndi] A/V thread for 'f3' started
00:46:12.116: [window-capture: 'Gimp'] update settings:
00:46:12.116: 	executable: gimp-2.10.exe
00:46:12.116: 	method selected: Automatic
00:46:12.116: 	method chosen: BitBlt
00:46:12.116: 
00:46:12.116: [window-capture: 'Chrome'] update settings:
00:46:12.116: 	executable: firefox.exe
00:46:12.116: 	method selected: Automatic
00:46:12.116: 	method chosen: WGC
00:46:12.116: 
00:46:12.116: [window-capture: 'Blender'] update settings:
00:46:12.116: 	executable: blender.exe
00:46:12.116: 	method selected: Automatic
00:46:12.116: 	method chosen: BitBlt
00:46:12.116: 
00:46:12.369: Switched to scene 'Digital 2sday 2'
00:46:12.369: ------------------------------------------------
00:46:12.369: Loaded scenes:
00:46:12.369: - scene 'ALL LIVE SCENES':
00:46:12.369:     - source: 'Closed Captions' (browser_source)
00:46:12.369:     - source: 'mics' (group)
00:46:12.369:         - source: 'Mod Mic Wireless' (wasapi_input_capture)
00:46:12.369:             - filter: 'Limiter' (limiter_filter)
00:46:12.369:             - filter: 'Compressor' (compressor_filter)
00:46:12.369:             - filter: 'Noise Suppression' (noise_suppress_filter)
00:46:12.369:             - filter: 'MODMIC Dedicated NDI™ output (Audio Only)' (ndi_audiofilter)
00:46:12.369:         - source: 'AT2020' (wasapi_input_capture)
00:46:12.369:             - filter: 'Limiter' (limiter_filter)
00:46:12.369:             - filter: 'Compressor' (compressor_filter)
00:46:12.369:             - filter: 'Noise Suppression' (noise_suppress_filter)
00:46:12.369:             - filter: 'Dedicated NDI™ output (Audio Only)' (ndi_audiofilter)
00:46:12.369:     - source: 'Digital 2sday 2' (scene)
00:46:12.369:     - source: 'Marker SYS [NS]' (scene)
00:46:12.369:     - source: 'Desktop Games [NS]' (scene)
00:46:12.369:         - filter: 'Desktop Games' (ndi_filter)
00:46:12.369:     - source: 'VR' (group)
00:46:12.369:         - source: 'VR game 2 CV  [NS]' (scene)
00:46:12.369:         - source: 'VR game 1 FPV  [NS]' (scene)
00:46:12.369:             - filter: 'VR game FPV' (ndi_filter)
00:46:12.369:     - source: 'Overlay banner [NS]' (scene)
00:46:12.369:     - source: 'Alerts  [NS]' (scene)
00:46:12.369:     - source: 'AUDIO PC & MIC [NS]' (scene)
00:46:12.369:     - source: 'Channel Point Memes [NS]' (scene)
00:46:12.369:     - source: 'Suva VUP AVATARS [NS]' (scene)
00:46:12.369:         - filter: 'Dedicated NDI™ output' (ndi_filter)
00:46:12.369:     - filter: 'All scenes' (ndi_filter)
00:46:12.369: - scene 'Countdown start [NS]':
00:46:12.369:     - source: 'black bg' (color_source)
00:46:12.369:     - source: 'Stream Starting soon and BRB [NS]' (scene)
00:46:12.369:     - source: 'COUNTDOWN' (ffmpeg_source)
00:46:12.369:         - monitoring: monitor and output
00:46:12.370:     - source: 'Emote Wall Widget start' (browser_source)
00:46:12.370:     - source: 'VLC Video Source' (vlc_source)
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370: - scene 'End credits':
00:46:12.370:     - source: 'Paul Adey LOL' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Elo Background BANNER' (image_source)
00:46:12.370:     - source: 'CREDITS' (browser_source)
00:46:12.370:         - monitoring: monitor only
00:46:12.370:         - filter: 'Chroma Key' (chroma_key_filter)
00:46:12.370:     - source: 'Emote Wall Widget start' (browser_source)
00:46:12.370: - scene '▼Nested Scenes▼':
00:46:12.370: - scene 'Alerts  [NS]':
00:46:12.370:     - source: 'Browser Alerts 2' (group)
00:46:12.370:         - source: 'Treat Stream' (browser_source)
00:46:12.370:             - monitoring: monitor and output
00:46:12.370:             - filter: 'Compressor' (compressor_filter)
00:46:12.370:         - source: 'Stream labs alerts' (browser_source)
00:46:12.370:             - monitoring: monitor and output
00:46:12.370:             - filter: 'Compressor' (compressor_filter)
00:46:12.370:             - filter: 'STREAMLABS ALERTS Dedicated NDI™ output (Audio Only)' (ndi_audiofilter)
00:46:12.370:     - source: 'Falcon Alert Server' (group)
00:46:12.370:     - source: 'falcon' (ndi_source)
00:46:12.370:     - source: 'f2' (ndi_source)
00:46:12.370:     - source: 'f3' (ndi_source)
00:46:12.370: - scene 'Animator chan pts':
00:46:12.370: - scene 'AT2020 MICROPONE [NS]':
00:46:12.370:     - source: 'AT2020' (wasapi_input_capture)
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:         - filter: 'Compressor' (compressor_filter)
00:46:12.370:         - filter: 'Noise Suppression' (noise_suppress_filter)
00:46:12.370:         - filter: 'Dedicated NDI™ output (Audio Only)' (ndi_audiofilter)
00:46:12.370: - scene 'AT2020 MIC status [NS]':
00:46:12.370:     - source: 'AT2020 MUTE SYM' (image_source)
00:46:12.370:     - source: 'AT2020 SYM' (image_source)
00:46:12.370: - scene 'AUDIO PC & MIC [NS]':
00:46:12.370:     - source: 'Music' (group)
00:46:12.370:         - source: 'VLC Video Source' (vlc_source)
00:46:12.370:             - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Audio 2 2' (group)
00:46:12.370:         - source: 'Mod Microphone [NS]' (scene)
00:46:12.370:         - source: 'AT2020 MICROPONE [NS]' (scene)
00:46:12.370:         - source: 'Desktop Audio' (wasapi_output_capture)
00:46:12.370:             - monitoring: monitor and output
00:46:12.370:             - filter: 'Limiter' (limiter_filter)
00:46:12.370:             - filter: 'Compressor' (compressor_filter)
00:46:12.370:             - filter: 'ELO PET Dedicated NDI AUDIO Output(Audio Only)' (ndi_audiofilter)
00:46:12.370: - scene 'Channel Point Memes [NS]':
00:46:12.370:     - source: 'ASMR' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Bonk Me' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Navi Eep Sub chat notification' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Owo' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Chan pts End Stream' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Change Avatar' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Truth or Dare' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:     - source: 'Highlight My Message' (ffmpeg_source)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370: - scene 'Black transition':
00:46:12.370: - scene 'Desktop Games [NS]':
00:46:12.370:     - source: 'Apex Legends' (game_capture)
00:46:12.370:     - filter: 'Desktop Games' (ndi_filter)
00:46:12.370: - scene 'Digital 2sday 2':
00:46:12.370:     - source: 'Display Capture' (monitor_capture)
00:46:12.370:     - source: 'Gimp' (window_capture)
00:46:12.370:     - source: 'Chrome' (window_capture)
00:46:12.370:     - source: 'Suva VUP AVATARS [NS]' (scene)
00:46:12.370:         - filter: 'Dedicated NDI™ output' (ndi_filter)
00:46:12.370:     - source: 'Desktop Audio' (wasapi_output_capture)
00:46:12.370:         - monitoring: monitor and output
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:         - filter: 'Compressor' (compressor_filter)
00:46:12.370:         - filter: 'ELO PET Dedicated NDI AUDIO Output(Audio Only)' (ndi_audiofilter)
00:46:12.370:     - source: 'Blender' (window_capture)
00:46:12.370: - scene 'LioranBoard tsl transmitter':
00:46:12.370:     - source: 'TSL Transmitter' (browser_source)
00:46:12.370: - scene 'Marker SYS [NS]':
00:46:12.370:     - source: 'Markers sys' (group)
00:46:12.370:         - source: 'Marker 0' (browser_source)
00:46:12.370:             - filter: 'Marker0' (ndi_audiofilter)
00:46:12.370:         - source: 'Marker1' (browser_source)
00:46:12.370:             - filter: 'Marker1' (ndi_audiofilter)
00:46:12.370:     - source: 'Elo Background BANNER' (image_source)
00:46:12.370: - scene 'Mod Mic status [NS]':
00:46:12.370:     - source: 'Modmic mute sym' (image_source)
00:46:12.370:     - source: 'Modmic sym' (image_source)
00:46:12.370: - scene 'Mod Microphone [NS]':
00:46:12.370:     - source: 'Mod Mic Wireless' (wasapi_input_capture)
00:46:12.370:         - filter: 'Limiter' (limiter_filter)
00:46:12.370:         - filter: 'Compressor' (compressor_filter)
00:46:12.370:         - filter: 'Noise Suppression' (noise_suppress_filter)
00:46:12.370:         - filter: 'MODMIC Dedicated NDI™ output (Audio Only)' (ndi_audiofilter)
00:46:12.370: - scene 'Overlay banner [NS]':
00:46:12.370:     - source: 'Overlay  Banner' (group)
00:46:12.370:         - source: 'banner over lay xq' (image_source)
00:46:12.370:             - filter: 'Color Correction' (color_filter)
00:46:12.370:         - source: 'Music note 1' (text_gdiplus)
00:46:12.370:             - filter: 'Color Correction' (color_filter)
00:46:12.370:         - source: 'Now Playing VLC LIVE' (text_gdiplus)
00:46:12.370:             - filter: 'Scroll' (scroll_filter)
00:46:12.370:             - filter: 'Color Correction' (color_filter)
00:46:12.370:         - source: 'Follow Count' (browser_source)
00:46:12.370:         - source: 'Music note 1 2' (text_gdiplus)
00:46:12.370:             - filter: 'Color Correction' (color_filter)
00:46:12.370:         - source: 'Social Media' (slideshow)
00:46:12.370:     - source: 'Bs INFO' (group)
00:46:12.370:         - source: 'Beat Saber Song info' (browser_source)
00:46:12.370:         - source: 'Beat Saber Song info 2' (browser_source)
00:46:12.370: - scene 'Stream Data [NS]':
00:46:12.398:     - source: 'Markers sys' (group)
00:46:12.398:         - source: 'Marker 0' (browser_source)
00:46:12.398:             - filter: 'Marker0' (ndi_audiofilter)
00:46:12.398:         - source: 'Marker1' (browser_source)
00:46:12.398:             - filter: 'Marker1' (ndi_audiofilter)
00:46:12.398:     - source: 'STREAM DATA' (group)
00:46:12.398:         - filter: 'Color Key' (color_key_filter)
00:46:12.398:         - source: 'Session Bit Rate' (browser_source)
00:46:12.398:         - source: 'Session Timer' (browser_source)
00:46:12.398:     - source: 'chat-box' (browser_source)
00:46:12.398:     - source: 'Stream Labs event list' (browser_source)
00:46:12.398:     - source: 'Activity-Feed' (browser_source)
00:46:12.398:         - filter: 'Chroma Key' (chroma_key_filter)
00:46:12.398:     - source: 'VR VIEW' (group)
00:46:12.398:         - source: 'VR CAM VIEW' (image_source)
00:46:12.398:         - source: 'VR FPV' (image_source)
00:46:12.398:     - source: 'Overlay banner [NS]' (scene)
00:46:12.398:     - source: 'Microphone Status' (group)
00:46:12.398:         - source: 'Mod Mic status [NS]' (scene)
00:46:12.398:         - source: 'AT2020 MIC status [NS]' (scene)
00:46:12.398:     - source: 'GAME BEING PLAYED' (group)
00:46:12.398:         - source: 'purple color' (color_source_v2)
00:46:12.398:         - source: 'Current Game txt' (text_gdiplus_v2)
00:46:12.398:         - source: 'twitch game category api' (browser_source)
00:46:12.398:     - filter: 'Dedicated NDI™ output Stream Data' (ndi_filter)
00:46:12.398: - scene 'Stream Starting soon and BRB [NS]':
00:46:12.398:     - source: 'Elo Background BANNER' (image_source)
00:46:12.398:     - source: 'BRB' (group)
00:46:12.398:         - source: 'Be Back Soon.' (text_gdiplus)
00:46:12.398:     - source: 'NOTIFICATIONS' (group)
00:46:12.398:         - source: 'NOW PLAYING VLC' (text_gdiplus)
00:46:12.398:             - filter: 'Scroll' (scroll_filter)
00:46:12.398:         - source: 'Twitch chat' (browser_source)
00:46:12.398:             - monitoring: monitor and output
00:46:12.398:         - source: 'note 2' (text_gdiplus)
00:46:12.398:         - source: 'note' (text_gdiplus)
00:46:12.398:     - source: 'Group' (group)
00:46:12.398:         - source: 'Recent event list' (browser_source)
00:46:12.398:             - monitoring: monitor and output
00:46:12.398:             - filter: 'Color Key' (color_key_filter)
00:46:12.398:             - filter: 'Color Correction' (color_filter)
00:46:12.398:         - source: 'Recent Events txt' (text_gdiplus)
00:46:12.398:         - source: 'Awesome ppl' (text_gdiplus)
00:46:12.398:         - source: '$UPPORT $croll' (text_gdiplus)
00:46:12.398:             - filter: 'Scroll' (scroll_filter)
00:46:12.398:         - source: 'Awesome ppl 2' (text_gdiplus)
00:46:12.398:         - source: 'Alerts  [NS]' (scene)
00:46:12.398:     - source: 'Music' (group)
00:46:12.398:         - source: 'VLC Video Source' (vlc_source)
00:46:12.398:             - filter: 'Limiter' (limiter_filter)
00:46:12.398: - scene 'Suva VUP AVATARS [NS]':
00:46:12.398:     - source: 'S.U.V.A' (game_capture)
00:46:12.398:         - filter: 'ELOPET OUT SUVA' (ndi_filter)
00:46:12.398:     - source: 'VUP' (game_capture)
00:46:12.398:         - filter: 'Color Key' (color_key_filter)
00:46:12.398:     - filter: 'Dedicated NDI™ output' (ndi_filter)
00:46:12.398: - scene 'Tip Cup [NS]':
00:46:12.398:     - source: 'Emote Wall Widget' (browser_source)
00:46:12.398:     - source: 'Stream Boss tip cup ' (group)
00:46:12.398:         - filter: 'Color Correction' (color_filter)
00:46:12.398:         - source: 'OwO' (image_source)
00:46:12.398:         - source: 'Heart' (image_source)
00:46:12.398:         - source: 'Stream Boss HP' (browser_source)
00:46:12.398:             - monitoring: monitor and output
00:46:12.398:             - filter: 'Chroma Key' (chroma_key_filter)
00:46:12.398:         - source: 'STREAM BOSS CUP' (image_source)
00:46:12.398:         - source: 'Recent event list' (browser_source)
00:46:12.398:             - monitoring: monitor and output
00:46:12.398:             - filter: 'Color Key' (color_key_filter)
00:46:12.398:             - filter: 'Color Correction' (color_filter)
00:46:12.398:         - source: 'Thank you' (text_gdiplus)
00:46:12.398: - scene 'VR GAMES [NS]':
00:46:12.398: - scene 'VR game 1 FPV  [NS]':
00:46:12.398:     - source: 'OpenVR Capture 2' (openvr_capture)
00:46:12.398:         - filter: 'VR HMD View' (ndi_filter)
00:46:12.398:     - source: 'vrchat ' (game_capture)
00:46:12.398:         - filter: 'Color Correction' (color_filter)
00:46:12.398:         - filter: 'Chroma Key' (chroma_key_filter)
00:46:12.398:         - filter: 'VRChat Dedicated NDI™ output' (ndi_filter)
00:46:12.398:     - filter: 'VR game FPV' (ndi_filter)
00:46:12.398: - scene 'VR game 2 CV  [NS]':
00:46:12.398:     - source: 'vrchat ' (game_capture)
00:46:12.398:         - filter: 'Color Correction' (color_filter)
00:46:12.398:         - filter: 'Chroma Key' (chroma_key_filter)
00:46:12.398:         - filter: 'VRChat Dedicated NDI™ output' (ndi_filter)
00:46:12.398:     - source: 'Game Capture' (game_capture)
00:46:12.398:     - source: 'OpenVR Capture 1' (openvr_capture)
00:46:12.398:         - filter: 'OpenVR Dedicated NDI™ output' (ndi_filter)
00:46:12.398: ------------------------------------------------
00:46:12.567: [obs-websocket] server started successfully on port 4444
00:46:12.578: [obs-websocket] io thread started
00:46:14.062: obs-browser: Access to XMLHttpRequest at 'https://api.twitch.tv/kraken/chat/emoticon_images?client_id=l6v6q35r9cy961a36pagn4freavkic6' from origin 'https://treatstream.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. (source: https://treatstream.com/notification/t/elocin_anagram/DNCSX0IZ/treat:0)
00:46:14.308: obs-browser: Uncaught (in promise) Something went wrong. (source: https://streamlabs.com/widgets/emote-wall?token=212A94FB5EB888E013B4:0)
00:46:14.308: obs-browser: Uncaught (in promise) Something went wrong. (source: https://streamlabs.com/widgets/emote-wall?token=212A94FB5EB888E013B4:0)
00:46:15.254: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:17.297: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:19.341: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:21.398: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:23.455: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:25.512: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:27.556: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:29.610: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:31.669: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:33.741: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:35.810: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:37.910: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:40.244: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:42.797: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:45.114: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:49.413: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:53.103: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:46:59.965: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:03.137: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:08.894: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:12.048: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:16.330: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:22.642: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:29.555: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:33.483: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:37.492: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:41.340: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:47.825: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:51.427: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:47:56.728: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:48:02.696: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:27.935: Last log entry repeated for 76 more lines
00:54:27.935: adding 42 milliseconds of audio buffering, total audio buffering is now 85 milliseconds (source: AT2020)
00:54:27.935: 
00:54:30.118: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:33.430: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:38.798: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:43.143: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:47.630: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:51.724: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:54:56.469: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:01.758: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:05.640: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:11.901: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:16.788: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:20.424: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:26.242: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:31.440: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:38.184: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:44.243: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:48.142: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:54.850: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:55:58.239: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:02.444: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:06.822: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:12.696: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:18.993: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:24.669: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:29.033: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:33.781: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:40.416: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:44.591: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:49.135: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:54.042: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
00:56:59.041: obs-browser: WebSocket connection to 'ws://127.0.0.1:9423/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED (source: file:///C:/Users/tarap/Videos/1-%20OBS/Stream%20Assets/lioranboard/tsl_transmitter.html:167)
```

</details>

## Steps to Reproduce

Open obs 
Start a recording.
Pause recording
Start a recording.
Pause recording
Start a recording.
Pause recording
Stop recording. 
(OBS stop recording process hangs.)
Close obs
Reopen OBS
(OBS says that there is an instance already running)
Check systray for obs icon with pause logo
ctrl+shift+esc to open task manager
OBS is not a listed application. 
check background processes
end obs running as a background process
It Will take several minutes but it will eventually end.

**Comment by WizardCM** [comment]
I want to say this issue is a duplicate of #2162 just with different steps.

**Comment by RytoEX** [comment]
Could you please retest this in OBS Studio 28.0.3 or newer, with and without third-party plugins? Without plugins is particularly important as we'd like to rule them out.

**Comment by RytoEX** [comment]
In OBS Studio 30.1.1, I can stop a paused recording without issue, meaning that I cannot get to the failed state indicated by the Steps to Reproduce. The core issue here is either the same as #2162 or was possibly fixed by commits since this report (#9900 possibly helped).
